apps/prairielearn/python/prairielearn/question_utils.py

```diff
@@ -96,7 +133,13 @@ def all_partial_scores_correct(data: QuestionData) -> bool:
 
 
 def add_files_format_error(data: QuestionData, error: str) -> None:
-    """Add a format error to the data dictionary."""
+    """Add a format error to the data dictionary.
+
+    Examples:
+        >>> add_files_format_error(data, f"Missing baz in foo.txt")
+        >>> data["format_errors"]
+        {"_files": ["Missing baz in foo.txt"]}
```

Comment:
Is this example worth including? It's just showing the contents of a dictionary that we're trying to abstract away from the client code.

Reply:
Given these are the reference docs, I think that they should show what the function actually does.